src/SonarQube.Client/Api/Requests/V5_10/GetIssuesRequest.cs

```diff
@@ -73,24 +81,7 @@ private static IEnumerable<T> ReadFromProtobufStream<T>(Stream stream, MessagePa
         }
 
         private static SonarQubeIssue ToSonarQubeIssue(ServerIssue issue) =>
-            new SonarQubeIssue(issue.Path, issue.Checksum, issue.Line, issue.Msg, issue.ModuleKey,
-                ParseResolutionState(issue.Resolution), issue.RuleKey);
-
-        private static SonarQubeIssueResolutionState ParseResolutionState(string resolution)
-        {
-            switch (resolution)
-            {
-                case "":
-                    return SonarQubeIssueResolutionState.Unresolved;
-                case "WONTFIX":
-                    return SonarQubeIssueResolutionState.WontFix;
-                case "FALSE-POSITIVE":
-                    return SonarQubeIssueResolutionState.FalsePositive;
-                case "FIXED":
-                    return SonarQubeIssueResolutionState.Fixed;
-                default:
-                    throw new ArgumentOutOfRangeException(nameof(resolution));
-            }
-        }
+            new SonarQubeIssue(issue.Path, issue.Checksum, issue.Line, issue.Msg, issue.ModuleKey, issue.RuleKey, 
+                issue.Status == "RESOLVED");
```

Comment:
It looks like the old API can return unresolved issues, and it also returns the _status_ field. Why are we hard-coding all issues to be resolved?